Can you fix the spacing to match the surrounding code?

Does this also fix this error?
```
In file included from ./include/SDL3/SDL_intrin.h:293:
/home/buildbot/buildslave/sdl3_android/build/android-ndk-r22/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/11.0.5/include/arm_sve.h:15:2: error: "SVE support not enabled"
#error "SVE support not enabled"
```

--- a/include/SDL3/SDL_intrin.h
+++ b/include/SDL3/SDL_intrin.h
@@ -291,8 +291,8 @@ _m_prefetch(void *__P)
 #  elif defined(SDL_PLATFORM_MACOS)
 /* Apple has no AArch64 device supporting SVE2 */
 #  elif defined(__ARM_ARCH) && (__ARM_ARCH >= 8) && (defined(__aarch64__) || defined(_M_ARM64))
-#      define SDL_SVE2_INTRINSICS 1
-#      include <arm_sve.h>
+#    define SDL_SVE2_INTRINSICS 1
+#    include <arm_sve.h>
 #  endif
 #endif
 #endif /* compiler version */

